lib/0001_retrieve-secrets.sh

```diff
@@ -68,12 +72,14 @@ def generate_vcap_creds_hash
     creds.has_key? key.to_s
   end
 
+  @creds_map.reject! { |key, _| creds[key.to_s].empty? }
```

Comment:
Is this line just intended to remove the cert from the return value if it's an empty string? Can you move it below the `valid_creds` conditional block that follows? It seems out-of-place here.

Reply:
yeah, I thought I already moved it. it should be set now.